pkg/control/control.go

```diff
@@ -54,6 +57,7 @@ func NewController(
 	nemesisGenerators []core.NemesisGenerator,
 	clientRequestGenerator func(ctx context.Context, client core.Client, node clusterTypes.ClientNode, proc *int64, requestCount *int64, recorder *history.Recorder),
 	verifySuit verify.Suit,
+	lokiCli *loki.LokiClient,
```

Comment:
How about use an interface here?

Reply:
I think it is ok to use lokiClient for now. But let me add a todo. What do you think